CHANGELOG.next.asciidoc

```diff
@@ -64,7 +64,6 @@
 - Fixed: limit for retries to Kibana configurable {issue-beats}[25063]
```

Comment:
Let's delete this entire file since we don't need it anymore. That will be a good hint to use the changelog tool instead.